### ISSUE TITLE
Include age of person whose birthday it is. The same for Anniversaries

### ISSUE THREAD

**Issue by mrsilver76** [plusonelabs/calendar-widget#79]
When you are subscribed to the "Contact's birthday and events" calendar then rather than stating:

`Fred Blogg's birthday`

it would be nice if it stated the age as well:

`Fred Blogg's 37th birthday`

If this causes problems with language translations, then:

`Fred Blogg's birthday (37)`

would also work.

**Comment by maiser2011** [comment]
Definitely a nice feature to have.  Would like to see it available.

**Comment by Offerel** [comment]
+1

**Comment by emucha** [comment]
Would be great for tracking ages of neices nephews etc. Please look into this further!

**Comment by lexa1** [comment]
good idea, please add this feature

**Comment by yvolk** [comment]
On how to see Google Calendar's birhdays and anniversaries (of Contacts) at all please read this comment in the related issue: https://github.com/plusonelabs/calendar-widget/issues/56#issuecomment-541054399

Regarding showing age I still don't know if these "calendars" have such information. Could anybody investigate this?

If not, the only solution will be integrating the Todo Agenda widget with actual Contacts, allowing the app to read them, and presenting birthdays and anniversaries from the Contacts as Events in the widget...

**Comment by virtualdj** [comment]
> Regarding showing age I still don't know if these "calendars" have such information. Could anybody investigate this?

No, they have not. You should **count the repetitions** like we discussed on #227 (sorry I didn't find this #79 at first otherwise I would have posted here).

> If not, the only solution will be integrating the Todo Agenda widget with actual Contacts, allowing the app to read them, and presenting birthdays and anniversaries from the Contacts as Events in the widget...

Other widgets (such as the one I named on #227) have an **additional "add-on" to install**, this way those who don't want a permission to access their contacts and don't need birthdays can simply avoid installing.
The add-on reads both the _Birth date_ and _Anniversary date_ field bound with the phone contacts and displays as a separate calendar on the main app (with a configurable color). Think as a "virtual calendar".
